CreeDictionary/cvd/__init__.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import re
+from functools import cache
+from os import fspath
+
+from gensim.models import KeyedVectors
+
+from CreeDictionary import settings
+from utils import shared_res_dir
+
+logger = logging.getLogger(__name__)
+
+shared_vector_model_dir = shared_res_dir / "vector_models"
+
+
+def _load_vectors(path):
+    return KeyedVectors.load(fspath(path), mmap="r")
+
+
+@cache
+def google_news_vectors():
+    return _load_vectors(shared_vector_model_dir / "news_vectors.kv")
+
+
+class DefinitionVectorsNotFoundException(Exception):
+    pass
+
+
+def definition_vectors_path():
+    filename = "definitions.kv"
+    if settings.USE_TEST_DB:
+        filename = "test_db_definitions.kv"
+    return shared_vector_model_dir / filename
+
+
+@cache
+def definition_vectors():
+    try:
+        return _load_vectors(definition_vectors_path())
+
+    except FileNotFoundError:
+        raise DefinitionVectorsNotFoundException
+
+
+# https://stackoverflow.com/a/48027864/14558
+def uniq(l: list) -> list:
```

Comment:
```suggestion
def unique(l: list) -> list:
```

Reply:
It’s named after the [unix shell command](https://linux.die.net/man/1/uniq).

Reply:
That doesn't mean unix is right...

Reply:
😂  :joy: :joy: kitâpwân!